_shared/project/Makefile

```diff
@@ -135,16 +108,19 @@ $(call help,make {{ target }},"run the functional tests in Python {{ python_vers
 	@pyenv exec tox -qe py{{ python_version|pyformat(PyFormats.MAJOR_MINOR_FMT) }}-functests
 
 {% endfor %}
-.PHONY: sure
-$(call help,make sure,"make sure that the formatting$(comma) linting and tests all pass")
-sure: python
-sure:
+.PHONY: backend-sure
+backend-sure: python
+backend-sure:
```

Comment:
This little `backend-sure` thing is to get `make sure` to run the backend stuff first, then the frontend stuff